traits: add support for CallExp when using getParameterStorageClasses (#12712)
dlang/dmd: changelog/trait-stc-callexp.md

# Function calls can now be used in `__traits(getParameterStorageClasses)`

> If parameter storage classes is needed from a function call, it is now possible

Reviewer: ```suggestion
Parameter storage classes can now be obtained from function calls as in this example:
```

Author: Done
